hera_qm/xrfi.py

```diff
@@ -70,14 +70,32 @@ def flag_xants(uv, xants, inplace=True):
         return uvo
 
 
+def resolve_xrfi_path(xrfi_path, file):
```

Comment:
I think it's considered bad style to call a variable `file`, since it's already a build in function.

Reply:
fixed